lib.ftd

```diff
@@ -3625,3 +3629,454 @@ align-self: start
+caption title:
+decimal monthly-price:
+decimal annually-price:
+optional string id: 
+
+-- ftd.column:
+width.fixed.percent: 33
+width if { ftd.device == "mobile" }: fill-container
+background.solid: $inherited.colors.background.step-1
+margin-horizontal.px if { ftd.device != "mobile" }: 16
+padding.px: 20
+border-radius.px: 10
+border-color: $inherited.colors.custom.two
+border-width.px:1
+
+-- ftd.text: $business-card.title
+role: $fonts.size-18
+color: $inherited.colors.text
+
+-- ftd.row:
+width: fill-container
+id: $business-card.id
+spacing.fixed.px: 4
+
+-- ftd.text: \$
+role: $inherited.types.heading-medium
+color: $inherited.colors.text
+
+-- ftd.decimal: 
+value if { change-prices }: $business-card.annually-price
+value: $business-card.monthly-price
+role: $inherited.types.heading-medium
+color: $inherited.colors.text
+
+-- end: ftd.row
+
+-- ftd.row:
+margin-top.px: 16
+margin-bottom.px: 39
+padding-vertical.px: 8
+padding-horizontal.px: 23
+align-content: center
+background.solid: $fonts.cdcdcd36
+width: fill-container
+spacing.fixed.px: 4
+
+-- ftd.image:
+src: $assets.files.static.pricing.dark-star.svg
+
+-- ftd.text:  Launching Soon 
+role: $inherited.types.copy-regular
+color: $inherited.colors.text
+
+-- end: ftd.row
+
+-- end: ftd.column
+
+-- end: business-card
+
+
+
+
+
+
+
+
+
+
+-- component business-plus-card:
+caption title:
+decimal monthly-price:
+decimal annually-price:
+optional string id: 
+
+-- ftd.column:
+width: fill-container
+width.fixed.percent if { ftd.device != "mobile" }: 33
+background.solid: $inherited.colors.border
+margin-horizontal.px if { ftd.device != "mobile" }: 16
+padding.px: 20
+border-radius.px: 10
+border-color: $inherited.colors.custom.two
+border-width.px:1
+
+-- ftd.column:
+anchor: parent
+background.solid: $fonts.005FBB
+width.fixed.px: 57
+height.fixed.px: 62
+top.px: 0
+right.px: 0
+border-bottom-left-radius.px: 10
+border-top-right-radius.px: 10
+align-content: center
+
+-- ftd.image:
+src: $assets.files.static.pricing.plus.svg
+
+-- end: ftd.column
+
+-- ftd.text: $business-plus-card.title
+role: $fonts.size-18
+color: $inherited.colors.background.step-1
+
+-- ftd.row:
+width: fill-container
+id: $business-plus-card.id
+spacing.fixed.px: 4
+
+-- ftd.text: \$
+role: $inherited.types.heading-medium
+color: $inherited.colors.background.step-1
+
+-- ftd.decimal: 
+value if { change-prices }: $business-plus-card.annually-price
+value: $business-plus-card.monthly-price
+role: $inherited.types.heading-medium
+color: $inherited.colors.background.step-1
+
+-- end: ftd.row
+
+-- ftd.row:
+margin-top.px: 16
+margin-bottom.px: 39
+padding-vertical.px: 8
+padding-horizontal.px: 23
+align-content: center
+background.solid: #cdcdcd36
+width: fill-container
+spacing.fixed.px: 4
+
+-- ftd.image:
+src: $assets.files.static.pricing.light-star.svg
+
+-- ftd.text: Launching Soon 
+role: $inherited.types.copy-regular
+color: $inherited.colors.background.step-1
+
+-- end: ftd.row
+
+-- end: ftd.column
```

Comment:
Add `role` and `color` properties to container instead of giving them individually to the child.

```suggestion
-- ftd.column:
width: fill-container
width.fixed.percent if { ftd.device != "mobile" }: 33
background.solid: $inherited.colors.border
margin-horizontal.px if { ftd.device != "mobile" }: 16
padding.px: 20
border-radius.px: 10
border-color: $inherited.colors.custom.two
border-width.px:1
color: $inherited.colors.background.step-1

-- ftd.column:
anchor: parent
background.solid: $fonts.005FBB
width.fixed.px: 57
height.fixed.px: 62
top.px: 0
right.px: 0
border-bottom-left-radius.px: 10
border-top-right-radius.px: 10
align-content: center

-- ftd.image:
src: $assets.files.static.pricing.plus.svg

-- end: ftd.column

-- ftd.text: $business-plus-card.title
role: $fonts.size-18

-- ftd.row:
width: fill-container
id: $business-plus-card.id
spacing.fixed.px: 4
role: $inherited.types.heading-medium

-- ftd.text: \$

-- ftd.decimal: 
value if { change-prices }: $business-plus-card.annually-price
value: $business-plus-card.monthly-price

-- end: ftd.row

-- ftd.row:
margin-top.px: 16
margin-bottom.px: 39
padding-vertical.px: 8
padding-horizontal.px: 23
align-content: center
background.solid: #cdcdcd36
width: fill-container
spacing.fixed.px: 4

-- ftd.image:
src: $assets.files.static.pricing.light-star.svg

-- ftd.text: Launching Soon 
role: $inherited.types.copy-regular

-- end: ftd.row

-- end: ftd.column
```